presto-docs/src/main/sphinx/functions/math.rst

```diff
@@ -130,6 +137,11 @@ Mathematical Functions
 
     Returns the constant Pi.
 
+.. function:: poisson_cdf(lambda, value) -> double
+
+    Compute the Poisson cdf with given lambda (mean) parameter:  P(N <= value; df).
+    The lambda parameter must be positive real numbers and value must be a non-negative integer.
```

Comment:
`must be a positive real number (of type DOUBLE)` or just say `must be a positive number of type DOUBLE`.